src/ISA.json

```diff
@@ -2,7 +2,7 @@
   "instructions": [
     {
       "mnemonic": "sha256sig1",
-      "name": "TBD",
+      "name": "Sigma1",
```

Comment:
For clarity, let's expand these names to the full instruction name (minus the redundant "instruction") as found in the "RISC-V Cryptography Extensions Volume I: Scalar & Entropy Source Instructions, section 2.6 "Zknh - NIST Suite: Hash Function Instructions" (edited for brevity):
| Mnemonic | Instruction |
| -- | -- |
| sha256sig0 | SHA2-256 Sigma0 instruction |
| sha256sig1 | SHA2-256 Sigma1 instruction |
| sha256sum0 | SHA2-256 Sum0 instruction |
| sha256sum1 | SHA2-256 Sum1 instruction |

So, this would be "SHA2-256 Sigma1", for example. Analogous changes for the other 3 instruction names in this PR, please.